http3/multipart.py

```diff
@@ -73,7 +79,9 @@ def render_data(self) -> bytes:
 
 def iter_fields(data: dict, files: dict) -> typing.Iterator[Field]:
     for name, value in data.items():
-        if isinstance(value, list):
+        if value is None:
```

Comment:
Let's not do the "omit None values" thing - I think we should just let `None` raise an error, same as any other non-str/bytes type.